packages/ui5-app/ui5.yaml

```diff
@@ -23,7 +23,9 @@ builder:
   - name: ui5-task-stringreplacer
     afterTask: replaceVersion
     configuration:
-      files: ["**/*.js", "**/*.xml"]
+      files:
+        - "**/*.js"
+        - "**/*.xml"
```

Comment:
can you please add some replacement configuration as described in the task's/middleware's README?

Reply:
Ack